app/src/main/java/com/example/numberbaseball/MainActivity.kt

```diff
@@ -8,9 +9,17 @@ class MainActivity : BaseActivity(R.layout.activity_main) {
 
     private val logger = Logger.getLogger(MainActivity::class.java.name)
 
-    override fun createAfterInit() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        viewInit()
+    }
+
+    /**
+     * 뷰 초기화
+     */
+    private fun viewInit() {
```

Comment:
함수명은 동사나 동사구가 되도록 명명해주세요.
```suggestion
    private fun initView() {
```
> **메서드 이름**
메서드 이름은 동사나 동사구가 적합하다. postPayment, deletePage, save 등이 좋은 예다.
접근자, 변경자, 조건자는 javabean 표준에 따라 값 앞에 get, set, is를 붙인다.

Reply:
[30f1304] 완료했습니다.